Blaise-Shyaka/Politico
# [#171720387] Create user signin API endpoint

# What does this PR do?

It creates an API endpoint allowing users to sign in to their existing accounts

# Description of the tasks to be completed

- Create a controller for the `/auth/signin`
- Mount it on a router
- Write tests for that endpoint

# How should this be manually tested?

- Clone this repository
- `npm install` all dependencies
- run `npm run test` in your terminal
- 7 test cases should pass

# What are the relevant pivotal tracker stories?

#171720387
